_maps/readme.md

```diff
@@ -4,29 +4,25 @@
 
 The list of items that are modular requiring adding/modifying to /tg/ maps:
 
-- Positronic Alert Consoles (inside Robotics area)
-- Command Vendor (inside Meeting area)
-- LustWish Vendor (inside Recreational/Dormitory area)
```

Comment:
I'm not sold on this one, mostly because I know people are going to complain a lot about having to go back to the interlink every time they want an item from it.

Reply:
It's not been on station for... a long while.

Reply:
What do you mean, it was on station right up until 2 days ago.

Reply:
I don't even remember the last time I reset it in this tedious process, so it's not me that has been doing it. But no complaints since it's on the interlink.

Reply:
Again, it's not a priority item, others can deal with it.